cmd/argocd-applicationset-controller/commands/applicationset_controller.go

```diff
@@ -95,6 +96,8 @@ func NewCommand() *cobra.Command {
 			cli.SetLogFormat(cmdutil.LogFormat)
 			cli.SetLogLevel(cmdutil.LogLevel)
 
+			ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Why add a new dependency? Can you reuse the existing logger impl?
 
```suggestion
			ctrl.SetLogger(logutils.NewLogrusLogger(logutils.NewWithCurrentConfig()))
```

Reply:
done